3.10/tutorials.md

```diff
@@ -98,6 +98,9 @@ Graphs
 - [k Shortest Paths Queries in AQL](https://www.arangodb.com/learn/graphs/k-shortest-paths-queries-in-aql/){:target="_blank"}:
   Query your graph for multiple shortest path alternatives.
 
+- [All Shortest Paths Queries in AQL](https://www.arangodb.com/learn/graphs/all-shortest-paths-queries-in-aql/){:target="_blank"}:
+  Query your graph for all shortest path alternatives.
```

Comment:
This can be removed if we decide not to mirror the following page for `ALL_SHORTEST_PATHS`: https://www.arangodb.com/learn/graphs/k-shortest-paths-queries-in-aql/